The FSF GCC Makefile does not install texinfo but GNUPro needs it
installed so that the info files can be built.

ChangeLog
2004-07-29  Nick Clifton  <nickc@redhat.com>

	* Makefile.def (module texinfo): Always install.
	* Makefile.in: Regenerate.


Index: Makefile.def
===================================================================
RCS file: /cvs/cvsfiles/gnupro/Makefile.def,v
retrieving revision 1.2
diff -c -3 -p -r1.2 Makefile.def
*** Makefile.def	29 Jan 2004 16:37:56 -0000	1.2
--- Makefile.def	29 Jul 2004 07:49:08 -0000
*************** host_modules= { module= shellutils; };
*** 77,83 ****
  host_modules= { module= sid; };
  host_modules= { module= sim; };
  host_modules= { module= tar; };
! host_modules= { module= texinfo; no_install= true; };
  host_modules= { module= textutils; };
  host_modules= { module= time; };
  host_modules= { module= uudecode; };
--- 77,83 ----
  host_modules= { module= sid; };
  host_modules= { module= sim; };
  host_modules= { module= tar; };
! host_modules= { module= texinfo; };
  host_modules= { module= textutils; };
  host_modules= { module= time; };
  host_modules= { module= uudecode; };
